core/env/dup_spec.rb

```diff
@@ -0,0 +1,11 @@
+require_relative '../../spec_helper'
+
+describe "ENV#dup" do
+  ruby_version_is "3.2" do
```

Comment:
**minor:** As far as I understand `ENV#dup` raises `TypeError` since Ruby 3.1, not 3.2